mumax3c/scripts/driver.py

```diff
@@ -17,7 +19,7 @@ def driver_script(driver, system, compute=None, **kwargs):
         mx3 += "tablesave()\n\n"
 
     if isinstance(driver, mc.RelaxDriver):
-        if mm.Damping() not in system.dynamics:
+        if not system.dynamics.get(type=mm.Damping):
             raise ValueError("A damping term is needed.")
         alpha = system.dynamics.damping.alpha
```

Comment:
```suggestion
        alpha = system.dynamics.get(type=mm.Damping)[0].alpha
```

Reply:
Thank you for pointing it out. I keep forgetting to replace these :/